### DISCUSSION TITLE
XWayland Programs Missing After Suspend

### DISCUSSION THREAD

**Discussion by ksolowoniuk** [hyprwm/Hyprland#3711] [Q&A] [answered]
I've been running Hyprland on FreeBSD 13 and everything was working fine until an update started causing some issues. I decided to try FreeBSD 14.0-RC3 but the problem persists. If I have Chromium running and either turn off my display or suspend with `acpiconf -s3` when I resume or turn the display back on the Chromium window is gone but `ps` still shows the process running. I tried running another XWayland application (xclock) and it also disappears but the process is gone too. If I `pkill chromium` and attempt to restart it Hyprland crashes. I'm really not sure if this is Hyprland specific but wasn't sure where else to ask for help.

**Comment by andresilva** [comment, marked as answer]
https://github.com/hyprwm/Hyprland/issues/3637 and https://github.com/hyprwm/Hyprland/issues/3422
